nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -31,6 +31,10 @@ with lib;
   # there is no power management backend such as upower).
   powerManagement.enable = true;
 
+  # Enable sound in graphical iso's.
+  hardware.pulseaudio.enable = true;
+  hardware.pulseaudio.systemWide = true; # Needed since we run plasma as root.
```

Comment:
Jikes.

Reply:
Another reason why I thought that running the graphical iso as root was not good https://github.com/NixOS/nixpkgs/pull/42610

Reply:
@worldofpeace May be worth to try to pick that one up again right after the branch-off for 19.03? :-)